packages/integration-tests/webpack.config.js

```diff
@@ -1,7 +1,21 @@
 const path = require('path');
+const { IgnorePlugin } = require('webpack');
 // path to module root
 const root = path.resolve(__dirname);
 
+const ignoredPackages = [
+  'mssql',
+  'mssql/lib/base',
+  'mssql/package.json',
+  'mysql',
+  'mysql2',
+  'oracledb',
+  'pg-native',
+  'pg-query-stream',
+  'sqlite3',
+  'tedious',
+];
+
```

Comment:
These changes to the webpack config were due to an issue importing PG models in /api/lib. The packages here (e.g. `integration-tests`) import the api package and needed this patch to be able to build.